resources/core/charts/gateway/templates/compass-gateway.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Can we get rid of this Gateway? I feel like we are complicating things a whole bunch instead of simplifying them. I guess the desired scenario would be to have Compass Gateways always in Compass chart and install them no matter if we are installing Compass with or without Core.
Can we achieve it?

Reply:
removed